security/pkg/pki/ca/ca.go

```diff
@@ -118,7 +124,7 @@ type IstioCAOptions struct {
 func NewSelfSignedIstioCAOptions(ctx context.Context,
```

Comment:
I think we are _well_ past due to make this a struct :slightly_smiling_face:

Reply:
Done